e2e/webpack/bundle-through/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@csstools/e2e--webpack--bundle-through",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Can you add postcss-preset-env to a bundle with Webpack?",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --config webpack.config.js",
+    "test": "node ./dist/bundle-from-mjs.cjs && node ./dist/bundle-from-cjs.cjs "
+  },
+  "dependencies": {
+    "postcss": "^8.4.6",
+    "webpack": "^5.68.0",
+    "webpack-cli": "^4.9.2",
+    "postcss-preset-env": "file:../../../plugin-packs/postcss-preset-env"
+  }
+}
```

Comment:
This is not part of the npm workspaces but it can be.

Instinctively I did not want to "pollute" the workspaces with dependencies for e2e tests.

I don't have a technical reason for this and it might make more sense to make it workspace.